{"name": "test_incorrect[incorrect_data0]", "status": "passed", "attachments": [{"name": "log", "source": "c038dc4c-3c96-444d-ac03-0237358fd0ff-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "incorrect_data", "value": "{'username': '1303845892@qq.com', 'password': '', 'check': '密码不能为空'}"}], "start": 1594723588957, "stop": 1594723591400, "uuid": "00c3c014-e9d8-4f29-a00b-ecaab59468e8", "historyId": "e64c0af6fef26d85bcdee262d7a7479c", "testCaseId": "486418aa0944ed0ecb75b40c2b58e369", "fullName": "TestCases.test_login.TestLogin#test_incorrect", "labels": [{"name": "tag", "value": "login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KFH4O7N"}, {"name": "thread", "value": "20556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}